src/components/GuideListView/GuideListView.tsx

```diff
@@ -191,11 +195,34 @@ export const GuideListView: FunctionComponent<GuideListViewProps> = ({
           handleUpdate={(tags) => setGuide({ ...guide, ...{ tags: tags } })}
         />
       </EuiFormRow>
+      <EuiFormRow
+        label="Custom URL Slug (optional)"
+        isInvalid={showErrors}
+        error={slugErrors}
+      >
+        <EuiFieldText
+          value={guide.slug !== guide._id ? guide.slug : ''}
+          onChange={(e) => {
+            if (
```

Comment:
Make a new function to handle this (handleSlugInput or something like that), we generally try to keep js code outside of the html